.github/workflows/pr-build.yml

```diff
@@ -25,9 +25,9 @@ jobs:
         with:
           ruby-version: 2.7
 
-      - name: Install and Build 🔧
-        run: |
-          bundle install
-          npm install
-          npm ls @loopback/docs 2>/dev/null || true
-          npm run build
+#      - name: Install and Build 🔧
+#        run: |
+#          bundle install
+#          npm install
+#          npm ls @loopback/docs 2>/dev/null || true
+#          npm run build
```

Comment:
I wonder why you commented the build step for the PR. The commands will NOT run with gh-pages branch and they should not mess up.

Reply:
you're right. At first, i thought we may not need this but I guess it's needed to make sure the web site can build. Will add it later.
I tested it again with the deploy action. https://github.com/dhmlau/loopback.io/blob/gh-pages/.github/workflows/site-build.yml#L35-L40. Will do more testing again on my repo first and update this PR.